.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
```

Comment:
Do we have lint rules for all of these? Particularly `insert_final_newline` feels like something we should lint for.

Reply:
I don't think any of these contradict our existing lint rules, but I expressly didn't want to include corresponding lint rules like that because it would affect a hilarious amount of files, and something like trailing new space is something we can progressively adopt over time and isn't important that it needs to even be a warning when linting.

Reply:
@seidior as long as this isn't in competition with anything we already have, I'm fine with it